vertica/datadog_checks/vertica/vertica.py

```diff
@@ -582,29 +558,8 @@ def get_connection(self):
             # but we still get logs via parent root logger
             connection_options['log_path'] = ''
 
-        if self._tls_verify:  # no cov
-            # https://docs.python.org/3/library/ssl.html#ssl.SSLContext
-            # https://docs.python.org/3/library/ssl.html#ssl.PROTOCOL_TLS
-            tls_context = ssl.SSLContext(protocol=PROTOCOL_TLS_CLIENT)
-
-            # https://docs.python.org/3/library/ssl.html#ssl.SSLContext.verify_mode
-            tls_context.verify_mode = ssl.CERT_REQUIRED
-
-            # https://docs.python.org/3/library/ssl.html#ssl.SSLContext.check_hostname
-            tls_context.check_hostname = self._validate_hostname
-
-            # https://docs.python.org/3/library/ssl.html#ssl.SSLContext.load_verify_locations
-            if self._cafile or self._capath:
-                tls_context.load_verify_locations(self._cafile, self._capath, None)
-
-            # https://docs.python.org/3/library/ssl.html#ssl.SSLContext.load_default_certs
-            else:
-                tls_context.load_default_certs(ssl.Purpose.SERVER_AUTH)
-
-            # https://docs.python.org/3/library/ssl.html#ssl.SSLContext.load_cert_chain
-            if self._cert:
-                tls_context.load_cert_chain(self._cert, keyfile=self._private_key)
-
+        if self._tls_verify:
```

Comment:
Let's use another config field for this called `use_tls` like in zk: https://github.com/DataDog/integrations-core/blob/master/zk/datadog_checks/zk/zk.py#L250